manifests/server/grant.pp

```diff
@@ -89,7 +89,7 @@
   # If you need such grant, use:
   # postgresql::grant { 'table:foo':
   #   role        => 'joe',
-  #   …
+  #   ...
```

Comment:
By teaching 'murican rednecks that foreign languages may use characters not listed in ascii? ;)